packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -302,9 +302,11 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 
 	const { sitePlan } = useSite();
 
-	const allAvailablePlans = [ defaultPaidPlan, nonDefaultPaidPlan, defaultFreePlan ].filter(
-		Boolean
-	) as Plan[];
+	const allAvailablePlans: ( Plan | undefined )[] = [
+		defaultPaidPlan,
+		...( nonDefaultPaidPlan ? [ nonDefaultPaidPlan ] : [] ),
+		defaultFreePlan,
+	];
```

Comment:
Small nit, it might be easier to read like this:
```suggestion
	const allAvailablePlans: ( Plan | undefined )[] = nonDefaultPaidPlan
		? [ defaultPaidPlan, nonDefaultPaidPlan, defaultFreePlan ]
		: [ defaultPaidPlan, defaultFreePlan ];
```